docroot/sites/admissions.uiowa.edu/modules/admissions_core/sass/pdf.scss

```diff
@@ -529,12 +528,17 @@ h6 {
     }
 
     thead td,
-    thead th {
-      background-color: $light;
+    thead th,
```

Comment:
We could probably move this back to Roboto.   Zilla is a smaller and heavier font and it is harder to read when it is rendered at a small size in all caps.